docs/blog/newsletter-november-2017.rst

```diff
@@ -0,0 +1,116 @@
+.. post:: November 05, 2017
+   :tags: newsletter
+
+
+#########################################
+Write the Docs Newsletter – November 2017
+#########################################
+
+Happy November everybody! Kelly O. here, reporting live from the coffee shop that is about to cut me off because I've had three flat whites in a row. But I'm considering them celebratory beverages, because we have a jam-packed month of documentation goodness to talk about.
+
+In the next few weeks, the orangization team will be:
+
+- Announcing the call for proposals (and looking for volunteers) for next year's Portland conference! (If you're not signed up to the conference mailing list, you can do so in `the sidebar of the WTD homepage <http://www.writethedocs.org/>`_.)
+- Wishing we were all going to be in Australia for their `Write the Docs Day mini-conference <http://www.writethedocs.org/conf/au/2017/>`_.
+- Looking on excitedly as API the Docs holds `their second one-day event in Amsterdam <http://apithedocs.org/>`_.
+- Musing on how we got so lucky as to have almost a dozen meetups this month in a cities across the western hemisphere. Scroll down for details!
+
+Of course, in the meantime, we also have an array of documentation stories below, for your reading pleasure. Enjoy!
+
+******************************
+Worth it: images & screenshots
+******************************
+This month, we had a couple of conversations about images and screenshots that raised some interesting questions. When do we need to include images in our documentation? How many images is enough? How many is too many? Given how difficult they can be to maintain, are they even worth it?
+
+Of course, our community of documentarians had a variety of good answers to share. For starters, one general sentiment was  that screenshots are inherently challenging. They take a significant time commitment to maintain, especially for a product with frequent UI changes. Speaking of screenshots, it was also pointed out that they are often overused and may not even be that useful to our readers. Diagrams, on the other hand, can be incredibly useful when designed well. Overall, the consensus was that using some screenshots and other images in your documentation is usually worth it, especially if you can commit some bandwidth to maintaining them.
```

Comment:
s/general// ?

Reply:
"Speaking of screenshots, " We already are, no?

Reply:
"Overall, the consensus" redundant?

Reply:
👍